sdk/go/client.go

```diff
@@ -0,0 +1,89 @@
+package ehsm
+
+import (
+    tls "crypto/tls"
+    "encoding/json"
+    "fmt"
+    http "net/http"
+    "sort"
+    "os"
+    "bytes"
+    "sync"
+
+    "github.com/iancoleman/orderedmap"
+)
+
+type Client struct {
+    modifyLock              sync.RWMutex
```

Comment:
any concern to use mutex?

Reply:
every function before use Client, same code in vault and aws sdk